### PR TITLE
update faucet documentation

### PR DESCRIPTION
**Pull request by xli**
## Motivation

Refresh document, and move it to better place.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### CODE REVIEW

File: client/faucet/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+| `amount`               | int    | Y         | amount of coins to mint                                              |
+| `auth_key`             | string | Y         | your account authentication key                                      |
+| `currency_code`        | string | Y         | the currency code, e.g. XDX                                          |
+| `return_txns`          | bool   | N         | returns the transactions for creating / funding the account          |
+| `is_designated_dealer` | bool   | N         | creates a designated dealer account instead of a parent VASP account |
+
+Notes:
+* By default, the account created is a parent VASP account.
+* Type bool means you set value to a string "true" or "false"
+* The service sbumits 1 transfer funds transactoin for an onchain account `auth_key`.
```
Comment (sunmilee) [inline client/faucet/README.md:28]:
```suggestion
* The service submits 1 transfer funds transaction for an onchain account `auth_key`.
```

**Review by sunmilee — APPROVED** [review]

File: client/faucet/README.md (hunk 1 above)
Comment (davidiw) [reply client/faucet/README.md:28]: I would change the way this sentence reads:

* For existing accounts as defined by the `auth_key`, the service submits 1 transfer funds transaction
Similarly the following sentence
* For new accounts as defined by the `auth_key`, the service first issues a transaction for creating the account and another for transferring funds.
* All funds transferred come from the account `000000000000000000000000000000dd`
* Clients should retry their request if the requests failed. One reason for failure is that, under load, the service may issue transactions with duplicate sequence numbers. Only one of those transactions will be executed, the rest will fail.

File: client/faucet/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* The service submits 2 transactions for a new `auth_key`, one for creating account, another for transferring funds from account `000000000000000000000000000000dd` to the created account.
+* The transactions created with account sequence number maybe duplicated for the parallel requests received on server, and only one of the transactions will be executed successfully. Client should retry the API call if it found the transactions submitted were failed.
+
+### Response
+
+If no query param `return_txns` or it is not "true", server returns an unsigned int 64 in HTTP response body. The number is the new account sequence number of the account `000000000000000000000000000000dd` on Testnet.
```
Comment (davidiw) [inline client/faucet/README.md:34]:
```suggestion
If no query param `return_txns` or it is not "true", server returns an unsigned int 64 in HTTP response body. The number is the account sequence number of the account `000000000000000000000000000000dd` on Testnet after executing the request.
```

File: client/faucet/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+When there is only one client calls to the API, this returned number can be used for looking up the transaction. But in concurrent use case, 2 requests may return same account sequence number.
```
Comment (davidiw) [inline client/faucet/README.md:35]:
```suggestion
Nominally, this number can be used for looking up the submitted transaction. However, under load, this number may be shared by multiple transactions.
```

**Review by davidiw — APPROVED** [review]
Thanks for updating documentation!!!!

**Pull request merged**